pages/tip.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { TipPage } from 'src/containers'
+
+export default function Tip() {
+  return (
+    <TipPage />
+  )
+}
```

Comment:
이렇게 바로 export 하는 방법도 있습니다~

```suggestion
export default TipPage;
```

Reply:
오 그렇네요! 코멘트 감사합니다!!